yb-voyager/cmd/root.go

```diff
@@ -99,20 +99,45 @@ func startPprofServer() {
 
 var noLockNeededList = []string{
 	"yb-voyager version",
+	"yb-voyager help",
+	"yb-voyager import",
+	"yb-voyager import data to",
 	"yb-voyager import data status",
+	"yb-voyager export",
+	"yb-voyager export data from",
 	"yb-voyager export data status",
+	"yb-voyager cutover",
 	"yb-voyager cutover status",
+	"yb-voyager initiate",
 	"yb-voyager initiate cutover to source",
 	"yb-voyager initiate cutover to target",
 	"yb-voyager initiate cutover to source-replica",
 	"yb-voyager end",
 	"yb-voyager end migration",
+	"yb-voyager archive",
+}
+
+var noPersistentPreRunNeededList = []string{
+	"yb-voyager version",
+	"yb-voyager help",
+	"yb-voyager import",
+	"yb-voyager import data to",
+	"yb-voyager export",
+	"yb-voyager export data from",
+	"yb-voyager initiate",
```

Comment:
How about the non-leaf commands under `initiate` e.g.
initiate
initiate cutover
initiate cutover to